kindlecomicconverter/KCC_gui.py

```diff
@@ -642,6 +650,10 @@ def changeDevice(self):
             self.modeChange(2)
         else:
             self.modeChange(1)
+        if profile['ForceColor']:
+            self.setColorMode(1)
+        else:
+            self.setColorMode(0)
```

Comment:
Why can't you just call called `GUI.colorBox.setChecked(true/false)` directly?

Reply:
That is indeed another option. I simply mimicked the other code (e.g. `modeChange()`). But if this is an issue, I can remove the new function and call `GUI.colorBox.setChecked()` directly.

Reply:
Yea just call it directly, ForceEnabled is already a book.

Reply:
Done. I pushed another commit for an easier review. This commit can get squashed later on.

Reply:
Whoops, meant to say `profile['ForceColor']` instead.

Also, does `setEnabled` do anything? The checkbox is already enabled.